test/integration/runner_common.cpp

```diff
@@ -21,13 +21,17 @@ namespace bp = boost::process;
 namespace bpt = boost::property_tree;
 namespace fs = std::filesystem;
 
+bool run_skipped_tests = false;
+
+namespace {
+
 std::string oidExe = OID_EXE_PATH;
 std::string configFile = CONFIG_FILE_PATH;
 
 bool verbose = false;
 bool preserve = false;
 bool preserve_on_failure = false;
-bool run_skipped_tests = false;
+std::string global_oid_args{};
```

Comment:
does boost split this correctly if this string contains multiple arguments, or is it expecting one argument per string in the vector?

Reply:
It does require one argument per string in the vector, so I've added in a call to `boost::split` for the env var (it didn't work for multiple arguments before)